spi/src/main/java/org/jboss/pnc/spi/BuildOptions.java

```diff
@@ -34,23 +34,25 @@
 @AllArgsConstructor
 @ToString
 @EqualsAndHashCode
-@Getter
```

Comment:
Why have you removed it from here? Lombok doesn't generate getter when there is already one present